### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by pkhasakia**

### CODE REVIEW

File: core_java/grep/log/grep.out [hunk 1]
```diff
@@ -0,0 +1 @@
+[    Is father, mother, Tybalt, Romeo, Juliet,, Enter Romeo and Juliet aloft, at the Window.,     And Romeo dead; and Juliet, dead before,,     Romeo, there dead, was husband to that Juliet;]
```
Comment (jarviscanada) [inline core_java/grep/log/grep.out]: git rm -r core_java/grep/log

File: profile/profile.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (jarviscanada) [inline profile/profile.json:1]: where is your profile in README.md format?

File: profile/profile.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Preferred name (Characters only)
+name: "Preet Khasakia"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_PreetKhasakia/
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: I am a recent graduate out of Ontario Tech University in Software Engineering. Throughout my years in undergrad, I was exposed to many different assignments and projects that allowed to gain knowledge in various aspects of programming essentials. Coursework in distributed systems, cloud computing and Capstone I/II had a great deal of hands-on experience with the ability to grow in a team-oriented work environment. I have recently joined Jarvis as a Data Engineer trainee to improve on my skillset and familiarize myself with real-world projects to continue to grow and apply knowledge throughout the program.
```
Comment (jarviscanada) [inline profile/profile.yaml]: summary: I am a recent graduate from Ontario Tech University in Software Engineering. Throughout my years in undergrad, I was exposed to many different assignments and projects that allowed me to gain knowledge in various aspects of programming essentials. Coursework in distributed systems, cloud computing and Capstone I/II had a great deal of hands-on experience with the ability to grow in a team-oriented work environment. I have recently joined Jarvis as a Software Engineer trainee to improve my skillset and familiarize myself with real-world projects to continue developing and applying knowledge throughout the program.
		
fixed some grammar

File: profile/profile.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    
+    description: "Designed and implemented a linux cluster agent to preserve and monitor real-time data of nodes running on CentOS. Application performs real-time data collection of hardware specifications and CPU memory usage. Created a Data Definition Language (DDL) script to hold specified data into tables used in the database. Modified 'host_info.sh' and 'host_usage.sh' to retrieve the necessary and insert it to the database."
+
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description: 
+        - "Grep App: Implemented a java application that mimics the Linux 'grep' command to search and retrieve matching strings from files. Technologies used for this project are Maven, Java, Docker as a deployment tool and IntelliJ IDE. Implementation of backend code uses Encapsulation along with an interface to provide a flexible and organized implementation of the application. Deployment is done by dockerizing the app which includes building an image locally, running the docker container and push the image onto Docker Hub for verification of the image."
```
Comment (jarviscanada) [inline profile/profile.yaml]: - "Grep App: Implemented a java application that mimics the Linux 'grep' command to search and retrieve matching strings from files. Technologies used for this project are Maven, Java, Docker as a deployment tool and IntelliJ IDE. Implementation of backend code uses Encapsulation and an interface to provide a flexible and organized application implementation. Deployment was done by dockerizing the app, which includes building an image locally, running the docker container, and pushing the image onto Docker Hub to verify the image." 

fixed grammar

File: profile/profile.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description: 
+        - "Grep App: Implemented a java application that mimics the Linux 'grep' command to search and retrieve matching strings from files. Technologies used for this project are Maven, Java, Docker as a deployment tool and IntelliJ IDE. Implementation of backend code uses Encapsulation along with an interface to provide a flexible and organized implementation of the application. Deployment is done by dockerizing the app which includes building an image locally, running the docker container and push the image onto Docker Hub for verification of the image." 
+        - "JDBC App: The JDBC application establishes a connection between the java Customer program to an RDBMS. The basic flow of the JDBC app follows a simple process of mounting the driver and creating a connection. From that connection comes executing SQL statements, and then commit/rollback and finally close the connection. The application consists of multiple java classes each with its own function of providing the methods and connections that allow the user to access database elements with SQL queries pre-defined within the Customer DAO class."
```
Comment (jarviscanada) [inline profile/profile.yaml]: - "JDBC App: The JDBC application establishes a connection between the java Customer program to an RDBMS. The basic flow of the JDBC app follows a simple process of mounting the driver and creating a connection. From that connection comes executing SQL statements, and then commit/rollback and finally closing the connection. The application consists of multiple java classes, each with its function of providing the methods and connections that allow the user to access database elements with SQL queries pre-defined within the Customer DAO class."
		
fixed grammar

File: profile/profile.yaml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+highlighted_projects:
+    #project name
+  - name: "IOT Smart Home Security System"
+    #URL is optional
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Built and established all hardware components to achieve a fully functioning security system. Contributed to develop a security system application, allowing users to remotely control the system. Integrated Amazon Alexa that features a voice command capability to remotely arm/disarm the system."
```
Comment (jarviscanada) [inline profile/profile.yaml]: description: "Built and established all hardware components to achieve a fully functioning security system. Contributed to developing a security system application, allowing users to control the system remotely. Integrated Amazon Alexa features a voice command capability to remotely arm/disarm the system."

File: profile/profile.yaml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  - name: "PizzaRia Cloud Application"
+    description: "Implemented custom location markers of an embedded Google Maps to show different store locations of PizzaRia. Incorporated a class to handle user inputs of updated selections made by a user, stored via RDS database on AWS. Implemented an emailing service for users to show the receipt of their order with location and order summary." 
+
+#List of professional experience
+#The job description allows hiring managers to understand your job and the technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+#The description must be more than 25 words. If your experience is not software related, please focus on soft skills and achievements (e.g. team collaboration, problem-solving, improvements, communication).
+professional_experience:
+  #Write your Jarvis role as a professional experience rather than training.
+  - title: "Data Engineer"
```
Comment (jarviscanada) [inline profile/profile.yaml]: Software Engineer

**Review by jarviscanada — APPROVED** [review]

File: core_java/grep/log/grep.out (hunk 1 above)
Comment (pkhasakia) [reply core_java/grep/log/grep.out]: Done

**Pull request merged**